corehq/apps/fixtures/views.py

```diff
@@ -489,17 +509,11 @@ def _upload_fixture_api(request, domain):
                     reverse('fixture_api_status', args=(domain, download_id))
                 )
 
-                curl_command = "curl -v --digest {} -u {}".format(
```

Comment:
I didn't see why this was implemented in https://github.com/dimagi/commcare-hq/pull/24025. no indication in the ticket or pr and it seemed a little strange for an api call